Install fails on linux systems

Installing packages currently fails when installing on a linux system. It looks like it may also fail on anything non-darwin, but I haven't tested it.

### Steps to reproduce:
Run `npm i` on a linux install. Resulting error is:
```
npm ERR! code EBADPLATFORM
npm ERR! notsup Unsupported platform for fsevents@1.2.9: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm ERR! notsup Valid OS:    darwin
npm ERR! notsup Valid Arch:  any
npm ERR! notsup Actual OS:   linux
npm ERR! notsup Actual Arch: x64

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/dan/.npm/_logs/2019-08-11T14_14_30_334Z-debug.log
```

**System Information**
npm: `6.10.3`
Node: `v10.16.0`
OS: `Ubuntu 18.04.2 LTS`

--- a/package.json
+++ b/package.json
@@ -110,7 +110,6 @@
     "enzyme-to-json": "^3.3.5",
     "file-loader": "^1.1.11",
     "fork-ts-checker-webpack-plugin": "^0.5.2",
-    "fsevents": "^1.2.9",
     "html-webpack-plugin": "^3.2.0",
     "jest": "^24.8.0",
     "jest-file": "^1.0.0",
@@ -127,6 +126,9 @@
     "webpack-bundle-tracker": "^0.3.0",
     "webpack-cli": "^3.3.6",
     "webpack-dev-server": "^3.7.2"
+  },
+  "optionalDependencies": {
+    "fsevents": "^1.2.9"
   },
   "babel": {
     "presets": [

